cmd/edgectl/aes_install.go

```diff
@@ -382,40 +386,11 @@ func (i *Installer) Perform(kcontext string) error {
 		return fmt.Errorf(noCluster)
 	}
 
-	// Allow overriding the source domain (e.g., for smoke tests before release)
-	manifestsDomain := "www.getambassador.io"
-	domainOverrideVar := "AES_MANIFESTS_DOMAIN"
-	overrideMessage := "Downloading manifests from override domain %q instead of default %q because the environment variable %s is set."
-	if amd := os.Getenv(domainOverrideVar); amd != "" {
-		i.ShowWrapped(fmt.Sprintf(overrideMessage, amd, manifestsDomain, domainOverrideVar))
-		manifestsDomain = amd
-	}
-
-	// Download AES manifests
-	crdManifests, err := getManifest(fmt.Sprintf("https://%s/yaml/aes-crds.yaml", manifestsDomain))
+	namespace, err := i.kubeinfo.Namespace()
 	if err != nil {
-		i.Report("fail_no_internet", ScoutMeta{"err", err.Error()})
-		return errors.Wrap(err, "download AES CRD manifests")
+		i.log.Printf("%w", err)
+		os.Exit(1)
```

Comment:
Why? Returning the error will result in an orderly shutdown.

Reply:
Yeah, well, just copy&pasted. Fixing...